cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/AbstractConductingEquipmentConversion.java

```diff
@@ -351,6 +350,32 @@ private EquivalentInjectionConversion getEquivalentInjectionConversionForDanglin
         }
     }
 
+    protected static EquivalentInjectionConversion getEquivalentInjectionConversionForAssembledDanglingLine(Context context, String boundaryNode, BoundaryLine boundaryLine) {
```

Comment:
What is your definition of assembled dangling lines? This is a wording really close to CGMES world, do you mean paired here?